bin/NativeTests/BigIntTest.h

```diff
@@ -0,0 +1,57 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+// This file contains stubs needed to make BigIntTest successfully compile and link as well
+// as a means to emulate behavior of objects that interact with BigInt class
+
+#include "..\..\lib\Common\Warnings.h"
+#include "..\..\lib\Common\Core\Api.cpp"
+#include "..\..\lib\Common\Common\NumberUtilities.cpp"
+
+namespace Js
+{
+    void Throw::FatalInternalError(long)
+    {
+        Assert("Not tested");
+    }
+
+    bool Throw::ReportAssert(char const *, unsigned int, char const *, char const *)
```

Comment:
This exists elsewhere as 
`bool Throw::ReportAssert(__in LPCSTR fileName, uint lineNumber, __in LPCSTR error, __in LPCSTR message)`

Why are you duplicating it? And why do you `Assert("Not tested")` which will always pass?

Reply:
I am mocking it. But you are right. I should assert(false) instead.

---
In reply to: [214491945](https://github.com/Microsoft/ChakraCore/pull/5657#discussion_r214491945) [](ancestors = 214491945)

Reply:
If you are stubbing it, then you'll need to match the prefast annotations with `__in` then or the build will complain as it is now. Why are you stubbing this out though? If these are invoked, then won't it fail just as hard as `Assert(false)` would?